test/e2e/create_test.go

```diff
@@ -0,0 +1,87 @@
+package e2e
+
+import (
+	"context"
+	"log"
+	"testing"
+	"time"
+
+	operator "github.com/openshift/windows-machine-config-operator/pkg/apis/wmc/v1alpha1"
+	wmc "github.com/openshift/windows-machine-config-operator/pkg/controller/windowsmachineconfig"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	instanceType        = "m4.large"
+	credentialAccountID = "default"
+	SSHKeyPair          = "libra"
+)
+
+func creationTestSuite(t *testing.T) {
+	noOfNodesToBeCreated := 1
+	t.Run("creation", func(t *testing.T) { testWindowsNodeCreation(t, noOfNodesToBeCreated) })
+	t.Run("ConfigMap validation", func(t *testing.T) { testConfigMapValidation(t, noOfNodesToBeCreated) })
+	t.Run("Secrets validation", func(t *testing.T) { testValidateSecrets(t, noOfNodesToBeCreated) })
+}
+
+// testWindowsNodeCreation tests the Windows node creation in the cluster
+func testWindowsNodeCreation(t *testing.T, nodeCount int) {
+	testCtx := framework.NewTestCtx(t)
+	namespace, err := testCtx.GetNamespace()
+	if err != nil {
+		t.Fatal(err)
+	}
+	// create WMCO custom resource
+	wmco := &operator.WindowsMachineConfig{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "WindowsMachineConfig",
+			APIVersion: "wmc.openshift.io/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "instance",
+			Namespace: namespace,
+		},
+		Spec: operator.WindowsMachineConfigSpec{
+			InstanceType: instanceType,
+			AWS:          &operator.AWS{CredentialAccountID: credentialAccountID, SSHKeyPair: SSHKeyPair},
+			Replicas:     nodeCount,
+		},
+	}
+	if err = framework.Global.Client.Create(context.TODO(), wmco,
+		&framework.CleanupOptions{TestContext: testCtx,
+			Timeout: cleanupTimeout, RetryInterval: cleanupRetryInterval}); err != nil {
+		t.Fatalf("error creating wcmo custom resource  %v", err)
+	}
+	// As per testing, each windows VM is taking roughly 12 minutes to be shown up in the cluster, so to be on safe
+	// side, let's make it as 60 minutes.
```

Comment:
What does `it` stand for here? I am guessing it is `timeout`. Please call that out. I also don't see where it is being set to 60 minutes.